docs/sources/operations/meta-monitoring/_index.md

```diff
@@ -54,20 +57,19 @@ All components of Loki expose the following metrics:
 
 Note that most of the metrics are counters and should continuously increase during normal operations.
 
-1. Your app emits a log line to a file that is tracked by Promtail.
-1. Promtail reads the new line and increases its counters.
-1. Promtail forwards the log line to a Loki distributor, where the received
+1. Your app emits a log line to a file that is tracked by Alloy.
+1. Alloy reads the new line and increases its counters.
+1. Alloy forwards the log line to a Loki distributor, where the received
    counters should increase.
 1. The Loki distributor forwards the log line to a Loki ingester, where the
    request duration counter should increase.
 
-If Promtail uses any pipelines with metrics stages, those metrics will also be
-exposed by Promtail at its `/metrics` endpoint. See Promtail's documentation on
-[Pipelines](https://grafana.com/docs/loki/<LOKI_VERSION>/send-data/promtail/pipelines/) for more information.
+If Alloy uses any pipelines with metrics stages, those metrics will also be
+exposed by Alloy at its `/metrics` endpoint.
 
 ### Metrics cardinality
 
-Some of the Loki observability metrics are emitted per tracked file (active), with the file path included in labels. This increases the quantity of label values across the environment, thereby increasing cardinality. Best practices with Prometheus labels discourage increasing cardinality in this way. Review your emitted metrics before scraping with Prometheus, and configure the scraping to avoid this issue.
+Some of the Loki observability metrics are emitted per tracked file (active), with the file path included in labels. This increases the quantity of label values across the environment, thereby increasing cardinality. Best practices with Prometheus labels discourage increasing cardinality in this way. The Kubernetes Monitoring Helm chart provides best practices for monitoring Loki, including how to manage cardinality, however it is important to be aware of cardinality when implementing auto scaling of components in your Loki cluster.
```

Comment:
Line 76 and line 82, can we change the double quotes around `metrics.go` to single quotes to match the rest of the formatting in this topic.